dataactbroker/handlers/fileHandler.py

```diff
@@ -397,14 +398,15 @@ def getStatus(self):
             submissionInfo["last_updated"] = submission.updated_at.strftime("%Y-%m-%dT%H:%M:%S")
 
             for job_id in jobs:
+                job = sess.query(Job).filter_by(job_id = job_id).one()
```

Comment:
Heh, yeah. This is happening in several places. I went ahead and refactored out the `getJobsBySubmission` function, which was returning a list of job_ids so that calling code could loop through said list and get the jobs. Now everyone just pulls the jobs.